fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -107,7 +107,7 @@ class GlobalTagThrottlerImpl {
 			if (opType == OpType::READ) {
 				readCost.setTotal(newCost);
 			} else {
-				writeCost.setTotal(CLIENT_KNOBS->GLOBAL_TAG_THROTTLING_RW_FUNGIBILITY_RATIO * newCost);
```

Comment:
Is the reason we delete the ratio here because the `newCost` is from `getWriteOperationCost` which already multiply the `GLOBAL_TAG_THROTTLING_RW_FUNGIBILITY_RATIO`?

Reply:
Yeah, that is the reason for not multiplying by this ratio again.